themes/default/content/learn/abstraction-encapsulation/component-resources/index.md

```diff
@@ -1,14 +1,13 @@
 ---
+title_tag: Building a Component Resource | Learn Pulumi
```

Comment:
similarly to comment above, the actual title tag ends up being `Abstraction & Encapsulation Overview | Learn Pulumi | Pulumi`.

Reply:
Hi Sean - I think this will be fine as well. I went ahead and double-checked that the new title tags will remain within the recommended character length (with the "| Pulumi" included), so we should be good here. Thank you!